public/pages/CreateIndex/components/IndexMapping/IndexMapping.tsx

```diff
@@ -0,0 +1,366 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { forwardRef, useCallback, useState, Ref, useRef, useMemo, useImperativeHandle } from "react";
+import { EuiTreeView, EuiIcon, EuiTreeViewProps, EuiButton, EuiSpacer, EuiButtonGroup, EuiLink } from "@elastic/eui";
+import { set, get, isEmpty } from "lodash";
+import JSONEditor, { IJSONEditorRef } from "../../../../components/JSONEditor";
+import { Modal } from "../../../../components/Modal";
+import { MappingsProperties, MappingsPropertiesObject } from "../../../../../models/interfaces";
+import CustomFormRow from "../../../../components/CustomFormRow";
+import MappingLabel, { IMappingLabelRef } from "../MappingLabel";
+import "./IndexMapping.scss";
+
+export const transformObjectToArray = (obj: MappingsPropertiesObject): MappingsProperties => {
```

Comment:
The constant functions in this file and others should be moved to a separate constants/helpers file.

Reply:
fixed on https://github.com/opensearch-project/index-management-dashboards-plugin/commit/a8922d1cdf96ce71675e7542abf9bb1e303a7110

Reply:
and on https://github.com/opensearch-project/index-management-dashboards-plugin/commit/92e7db4f89ff5c00d3d5013c7a8e1756f3c9162c